electrum/gui/qt/main_window.py

```diff
@@ -2119,7 +2134,10 @@ def label(idx, ks):
             vbox.addWidget(mpk_text)
 
         vbox.addStretch(1)
-        btns = run_hook('wallet_info_buttons', self, dialog) or Buttons(CloseButton(dialog))
+        buttons = Buttons(CloseButton(dialog))
```

Comment:
Think about using ternary operator.

Reply:
Do you mean sth like `btns = self.wallet_info_notification_buttons(dialog) if self.wallet_info_notification_buttons else run_hook('wallet_info_buttons', self, dialog) or Buttons(CloseButton(dialog))` ?

Reply:
Yup

Reply:
I am wondering if we don't lose on legibility of the code